modules/ROOT/pages/changes-deprecations-removals.adoc

```diff
@@ -0,0 +1,289 @@
+
+|<prefix>.causal_clustering.read_replica.pull_update_highest_tx_id_received
+
+|===
+====
+
+=== SSL framework
+
+.See the CBC-based ciphers that are no longer used from Neo4j 2025.01:
+
+* TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384
+* TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256
+* TLS_DHE_RSA_WITH_AES_256_CBC_SHA256
+* TLS_DHE_RSA_WITH_AES_128_CBC_SHA256
+
+For more details, refer to the page on link:{neo4j-docs-base-uri}/operations-manual/5/security/ssl-framework/#ssl-other-configs[SSL framework] under the Security section.
+
+.Private keys
+
+Versions before Neo4j 5.0 allow keys to be stored with the old PKCS #1 standard.
+You can identify them by the line `-----BEGIN RSA PRIVATE KEY-----` at the beginning of the file.
+While Neo4j 5.0 can load and use those keys, they are considered deprecated and will be removed in a future version.
+
+
+.SSL-related configuration setting (or move it to the section 'Changed default values'?)
+
+Starting from 2025.01, the default value of `dbms.ssl.policy.*.verify_hostname` is changed from `false` to `true`.
+See details in the table link:{neo4j-docs-base-uri}/operations-manual/5/security/ssl-framework/#ssl-configuration[in the table under Configuration section on the page SSL framework].
+
+=== Ports
+
+Port `5000` is no longer used from 2025.01 onwards.
+
+=== Deprecated built-in procedures
+
+.See all deprecated procedures
+[%collapsible]
+====
+[options=header, cols="3m,1,1,3"]
+|===
+| Name
+| Community Edition
+| Enterprise Edition
+| Comment
+
+| link:{neo4j-docs-base-uri}/operations-manual/5/procedures/#procedure_dbms_quarantineDatabase[`dbms.quarantineDatabase()`] label:admin-only[]
+|
+| {check-mark}
+| Replaced by xref:procedures.adoc#procedure_dbms_unquarantineDatabase[`dbms.unquarantineDatabase()`]
+|===
+====
+
+=== Java API
+
+In 2025.01, the deprecated public API is removed.
+
+=== Neo4j configuration settings with changed default values
+
+[options=header, cols="3m,1,1"]
+|===
+| Name
+| Old default value
+| New default value
+
+|`dbms.cypher.infer_schema_parts`
+|`OFF`
+|`MOST_SELECTIVE_LABEL`
+
+|`db.logs.query.annotation_data_format`
+|`CYPHER`
+|`JSON`
+
+|`dbms.databases.seed_from_uri_providers`
+|`S3SeedProvider,CloudSeedProvider`
+|`CloudSeedProvider`
+
+|`server.metrics.csv.rotation.compression`
+|`NONE`
+|`ZIP`
+
+|`server.panic.shutdown_on_panic`
+|`false`
+|`true`
+
+|===
+
+
+=== Changed valid values of configuration settings
+
+The `server.logs.config` is changed to depend on the provided path for the _conf_ directory instead of having the _conf_ directory encoded in the default values.
+
+[options=header, cols="2m,3,3"]
+|===
+| Name
+| Old valid value
+| New valid value
+
+|`server.logs.config`
+|A path. If relative, it is resolved from `server.directories.neo4j_home`.
+|A path. If relative, it is resolved from `server.directories.configuration`.
+|===
+
+=== Renamed configuration settings
+
+`dbms.cluster.discovery.endpoints` => `dbms.cluster.endpoints`
```

Comment:
@nick-giles-neo, is this setting is renamed or the `dbms.cluster.discovery.v2.endpoints`?